docs/api/server/Environment-Variables.md

```diff
@@ -94,6 +96,24 @@ One App can be configured via Environment Variables:
 
 > ⚠️ = Required
 
+## `DOTENV_CONFIG_PATH`
+
+One App supports the usage of [Dotenv](https://github.com/motdotla/dotenv). This environment variable allows you to specify the path to the `.env` file that should be loaded. These values are loaded before the start of the One App server. See the documentation for Dotenv for advanced usage.
```

Comment:
Can you mention the default value?

The above makes it seem like if this env var is not passed dotenv won't load .env, but it will?

Reply:
Done [ce55733](https://github.com/americanexpress/one-app/pull/1333/commits/ce5573356dca8cb57785a7b10ea95de783244adc)